src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.PreserveCompilationContext.targets

```diff
@@ -32,7 +32,7 @@ Copyright (c) .NET Foundation. All rights reserved.
         <Optimize>$(Optimize)</Optimize>
         <AssemblyOriginatorKeyFile>$(AssemblyOriginatorKeyFile)</AssemblyOriginatorKeyFile>
         <DelaySign>$(DelaySign)</DelaySign>
-        <PublicSign>$(DelaySign)</PublicSign>
+        <PublicSign>$(PublicSign)</PublicSign>
```

Comment:
deleting this line would mean the same thing?

Reply:
I do not know why somebody wanted to preserve `PublicSign` sign here. If you know for sure that it does not make sense to preserve `PublicSign` here, the line can be deleted.

Reply:
you are right. this is under `DependencyFileCompilerOptions`, not a regular `PropertyGroup`.